src/router_core/route_tables.c

```diff
@@ -784,19 +852,19 @@ static void qdr_do_link_lost(qdr_core_t *core, qdr_general_work_t *work)
 }
 
 
-void qdr_post_mobile_added_CT(qdr_core_t *core, const char *address_hash, qd_address_treatment_t treatment)
+void qdr_post_set_mobile_seq_CT(qdr_core_t *core, int router_maskbit, uint64_t mobile_seq)
 {
-    qdr_general_work_t *work = qdr_general_work(qdr_do_mobile_added);
-    work->field = qdr_field(address_hash);
-    work->treatment = treatment;
+    qdr_general_work_t *work = qdr_general_work(qdr_do_set_mobile_seq);
+    work->in_conn_id = mobile_seq;
```

Comment:
I see what you did there.  Bad Ted: no soup for you!

Reply:
Ok... You're right...  I fixed it.